lcov.info

```diff
@@ -7,21 +7,27 @@ DA:10,1
 DA:12,1
 DA:13,1
 DA:18,1
-DA:19,1
-DA:21,1
-DA:27,1
-LF:11
-LH:11
+DA:20,1
```

Comment:
It's for code climate [test coverage](https://codeclimate.com/github/Ticketfly/ember-hook/coverage). That being said, I'm not sure if we need to include this ourselves, or if Travis will create this file during the CI process. I'll look into this more when I get in.